src/coreclr/inc/clrconfigvalues.h

```diff
@@ -572,11 +572,16 @@ RETAIL_CONFIG_DWORD_INFO(INTERNAL_HillClimbing_GainExponent,
 #define TC_CallCountThreshold (2)
 #define TC_CallCountingDelayMs (1)
 #define TC_DelaySingleProcMultiplier (2)
+#define TC_DelayPgoDivider (1)
```

Comment:
NOTE: this divider is needed for `R2R -> tier0inst -> tier1` case because once we switch to tier0 that doesn't inline code - we start to raise a lot of "first-time" compilations each of which delays promotion for the pendingTier1 methods in the background (by 100ms). However, when we do `R2R -> tier1inst ->tier1` (with `DOTNET_TC_InstrumentOptimizedCode=1` - we're fine even with the existing number.